src/lib/components/layout/v2/panels.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface Ratio {
+  left: number;
+  right: number;
+}
+
+const ThirtySeventy: Ratio = {
+  left: 0.3,
+  right: 0.7,
+};
+
+const SeventyThirty: Ratio = {
+  left: 0.7,
+  right: 0.3,
+};
+
+const FiftyFifty: Ratio = {
+  left: 0.5,
+  right: 0.5,
+};
+
+export const PanelsRatios = {
+  ThirtySeventy,
+  SeventyThirty,
+  FiftyFifty,
+};
+
+const PanelsContainer = styled.div<{
+  hasPadding?: boolean;
+}>`
+  display: flex;
+  flexdirection: row;
+  flex: 1;
+  padding: ${(props) => (props.hasPadding ? '1rem' : 'initial')};
+`;
+
+const LeftPanel = styled.div<{ hasPadding?: boolean; flex?: number }>`
+  flex: ${(props) => props.flex || '0 auto'};
+  display: flex;
+  padding: ${(props) => (props.hasPadding ? '1rem' : 'initial')};
+`;
+
+const RightPanel = styled.div<{ hasPadding?: boolean; flex?: number }>`
+  flex: ${(props) => props.flex || '0 auto'};
+  display: flex;
+  padding: ${(props) => (props.hasPadding ? '1rem' : 'initial')};
+`;
+
+export const TwoPanelLayout = ({
+  panelsRatio,
```

Comment:
I would better opt for a constrained `ratio` props that takes those values: `30-70` `50-50` `70-30`. it will be better to strongly enforce the ratios across the application.